lib/utils/utils.js

```diff
@@ -121,3 +122,25 @@ utils.writeMessage = function ( prompt, message ) {
   }
   prompt.write( message );
 };
+
+
+/**
+ * Helper for getting the length of a string containing unicode symbols
+ * From https://github.com/mathiasbynens/esrever/blob/master/scripts/export-data.js
+ *
+ * @param {String} input  - The unicode string
+ * @return {Number}       - The string length
+ */
+
+var REGEX_SYMBOLS = /([\0-\u02FF\u0370-\u1DBF\u1E00-\u20CF\u2100-\uD7FF\uDC00-\uFE1F\uFE30-\uFFFF]|[\uD800-\uDBFF][\uDC00-\uDFFF]|[\uD800-\uDBFF])([\u0300-\u036F\u1DC0-\u1DFF\u20D0-\u20FF\uFE20-\uFE2F]+)/g;
```

Comment:
Most linters can be configured to ignore lines with regular expressions in, for that exact reason. You can use the constructor function, but then you're compromising code quality, which isn't great.